Source/WebKit/UIProcess/API/glib/WebKitInstallMissingMediaPluginsPermissionRequest.cpp

```diff
@@ -51,14 +41,13 @@ using namespace WebCore;
  * it is allowed by default.
  *
  * Since: 2.10
+ *
+ * Deprecated: 2.40
  */
```

Comment:
Would be good to update this documentation to indicate that it no longer does anything, in addition to marking it as deprecated. Something like:

```
/**
 * WebKitInstallMissingMediaPluginsPermissionRequest:
 * @See_also: #WebKitPermissionRequest, #WebKitWebView
 *
 * Previously, a permission request for installing missing media plugins.
 *
 * WebKitInstallMissingMediaPluginsPermissionRequest will no longer ever be created, so
 * you can remove any code that attempts to handle it.
 *
 * Since: 2.10
 *
 * Deprecated: 2.40
 */
```